### DISCUSSION TITLE
i want to get ride of custom colors, make dark static and only keep datagrid view

### DISCUSSION THREAD

**Discussion by hxmus** [perspective-dev/perspective#2812] [Q&A] [answered]
i want to keep those by default, i tried to do it via removing elements atfer they are rendered but it's so ugly.

![firefox_kXbInSl9zo](https://github.com/user-attachments/assets/ed3343c8-4fdf-4ae7-b1be-55f5dd1f3507)

**Comment by texodus** [comment, marked as answer]
[Theme configuration (with examples) is covered in the 3rd paragraph of the docs](https://docs.rs/perspective-viewer/latest/perspective_viewer/#theming).
